src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java

```diff
@@ -373,7 +421,7 @@ public void scan(HttpMessage msg, String param, String value) {
                     || errorMatcher.find()) {
 
                 if (log.isDebugEnabled()) {
-                    log.debug("It IS possible to check for local file Path Traversal on the url filename on ["
+                    log.debug("It is possible to check for local file Path Traversal on the url filename on ["

```

Comment:
This was an original typo with caps (my fault done a bit of time ago)... this should be lowercase ;-)